HACKING.md

```diff
@@ -54,6 +54,44 @@ See the [Testing guide](TESTING.md).
 
 Given that the `--debug` option in snapcraft is reserved for project specific debugging, enabling for the `logger.debug` calls is achieved by setting the "SNAPCRAFT_ENABLE_DEVELOPER_DEBUG" environment variable to a truthful value. Snapcraft's internal tools, e.g.; `snapcraftctl` should pick up this environment variable as well.
 
+
+## Documentation
+
+
+### Build
+
+To render the documentation as HTML in `docs/_build`, run:
+
+```shell
+tox run -e build-docs
+```
+
+If you prefer to compose pages interactively, you can host the documentation on a local server:
+
+```shell
+tox run -e autobuild-docs
+```
+
+You can reach the interactive site at http://127.0.0.1:8080 in a web browser.
+
+
+### Test
+
+The documentation Makefile provided by the [Sphinx Starter Pack](https://github.com/canonical/sphinx-docs-starter-pack) provides a number of natural language checks such as style guide adherence, inclusive words, and product terminology, however they currently aren't configured correctly for Snapcraft. Instead, you can validate for basic language and syntax using two of the development tests.
+
+To check for syntax errors in documentation, run:
+
+```shell
+tox run -e lint-docs
+```
+
+For a rudimentary spell check, you can use codespell:
+
+```shell
+tox run -e codespell
```

Comment:
```suggestion
tox run -e lint-codespell
```

You can (but needn't) also point out the `format-codespell` tax environment, which will automatically replace misspellings if it only has one suggestion.